forch/forchestrator.py

```diff
@@ -37,7 +37,10 @@
 _BEHAVIORAL_CONFIG_DEFAULT = 'faucet.yaml'
 _SEGMENTS_VLAN_DEFAULT = 'segments-to-vlans.yaml'
 _DEFAULT_PORT = 9019
-_PROMETHEUS_HOST = '127.0.0.1'
+_FAUCET_VARZ_HOST = '127.0.0.1'
+_FAUCET_PROM_PORT = 9302
```

Comment:
these are defaults not actuals (name is misleading)

Reply:
done